### ISSUE TITLE
udevadm: -y documentation wrong

### ISSUE THREAD

**Issue by rstuart** [systemd/systemd#6792]
Bug report

Forwarding https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=865081

Package: udev
Version: 232-25
Severity: normal
Tags: patch

In short, -y (--sysname-match) matches the last component of the filename.  The manual page for udevadm says it matches a "sys device path".

Longer rant: after passing every possible way of writing a "sys device path" to the bloody thing, (eg /sys/devices/pci0000:00/0000:00:1c.0/0000:02:00.0/net/wlp2s0, /sys/devices/pci0000:00/0000:00:1c.0/0000:02:00.0/* */pci0000:00/0000:00:1c.0/0000:02:00.0/*, ...) I gave up and looked up the source.  The relevant line in src/libsystemd/sd-device/device-enumerator.c:498 is:

                if (!match_sysname(enumerator, dent->d_name))
			continue;
Sigh.

Suggested patch:

```
--- udevadm.xml	
2016-11-04 03:16:42.000000000 +1000
+++ udevadm.xml.new	2017-06-19 15:41:49.413529193 +1000
@@ -306,10 +306,11 @@
         </varlistentry>
         <varlistentry>
           <term><option>-y</option></term>
-          <term><option>--sysname-match=<replaceable>PATH</replaceable></option></term>
+          <term><option>--sysname-match=<replaceable>FILENAME</replaceable></option></term>
           <listitem>
-            <para>Trigger events for devices with a matching sys
-            device path. This option can be specified multiple times
+	    <para>Trigger events for devices with a matching
+	    the last component (the filename) of a sys device path.
+	    This option can be specified multiple times
             and supports shell style pattern matching.</para>
           </listitem>
         </varlistentry>
```

**Comment by poettering** [comment]
Any chance you you submit the patch as PR? if so we can review/merge it using the usual workflow! thanks!

**Comment by rstuart** [comment]
Done.

**Comment by poettering** [comment]
Fix waiting in #6809.
